autogluon/task/tabular_prediction/predictor.py

```diff
@@ -254,6 +254,8 @@ def fit_summary(self, verbosity=3):
             print("*** Summary of fit() ***")
             print("Estimated performance of each model:")
             results['leaderboard'] = self._learner.leaderboard(silent=False)
+            results['leaderboard']['fit_time'] = results['leaderboard']['fit_time'].fillna(0)
+            results['leaderboard']['pred_time_val'] = results['leaderboard']['pred_time_val'].fillna(0)
```

Comment:
since you mentioned you didn't want to contaminate the stored statistics with wrong data, you could consider moving the NA imputation into the plotting function, since that's the only place where it's needed:

https://github.com/awslabs/autogluon/blob/master/autogluon/utils/plots.py#L92-L93

Reply:
Will submit separate PR for moving fillna calls to plotting